test/src/module/command/archivePushTest.c

```diff
@@ -650,7 +783,7 @@ testRun(void)
             strZ(
                 strNewFmt(
                     "P00   INFO: push 2 WAL file(s) to archive: 000000010000000100000001...000000010000000100000002\n"
-                    "P01   WARN: WAL file '000000010000000100000001' already exists in the archive with the same checksum\n"
+                    "P01   WARN: WAL file '000000010000000100000001' already exists in the repo1 archive with the same checksum\n"
                     "            HINT: this is valid in some recovery scenarios but may also indicate a problem.\n"
                     "P01 DETAIL: pushed WAL file '000000010000000100000001' to the archive\n"
                     "P01   WARN: could not push WAL file '000000010000000100000002' to the archive (will be retried): "
```

Comment:
Generally speaking the error message that follows the warning should contain enough information to figure out where the error is happening, but I agree that it might be nice to make it more explicit.